src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+import { getNumbers } from './../../utils';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (event: React.MouseEvent<HTMLAnchorElement>) => void;
+}
+
+const Pagination: React.FC<PaginationProps> = props => {
+  const { total, perPage, currentPage, onPageChange } = props;
+  const pageItemsCount = Math.ceil(total / perPage);
+  const fromWhatItem = (currentPage - 1) * perPage + 1;
+  let toWhatItem = fromWhatItem + perPage - 1;
+
+  if (toWhatItem > total) {
+    toWhatItem = total;
+  }
+
+  const items = getNumbers(fromWhatItem, toWhatItem).map((n: number) => (
+    <li data-cy="item" key={'item' + n}>
+      Item {n}
+    </li>
+  ));
+  const pageItems = getNumbers(1, pageItemsCount).map((n: number) => (
+    <li
+      className={cn('page-item', { active: currentPage === n })}
+      key={'key' + n}
+    >
+      <a data-cy="pageLink" className="page-link" onClick={onPageChange}>
```

Comment:
The `onClick` handler for the page links should pass the page number to the `onPageChange` function. Currently, it only passes the event object, which might not be sufficient for changing the page correctly.